plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.ui.workbench,
- org.eclipse.ui
+ org.eclipse.ui,
+ org.eclipse.search
```

Comment:
Filing an issue sounds like a good idea.

It's a close call, but I prefer copying the image since it also removes the extra if to choose between GIF and PNG; but up to you.